skimage/io/tests/test_imageio.py

```diff
@@ -9,26 +9,20 @@
 from skimage._shared.testing import (assert_array_almost_equal, TestCase,
                                      expected_warnings)
 
+from pytest import importorskip
 
-try:
-    import imageio as _imageio
-except ImportError:
-    imageio_available = False
-else:
-    imageio_available = True
+importorskip('imageio')
```

Comment:
Why is this a thing? We depend on imageio

Reply:
maybe one day we won't ;) but ok.

Reply:
This was removed.